Makefile

```diff
@@ -1,39 +1,66 @@
 #!/usr/bin/make -f 
-DOCKER_COMPOSE ?= docker-compose
-src_dir        := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
+DOCKER_COMPOSE			?= docker-compose
+DOCKER_IMAGE_REPO		?= seznam/slo-exporter
+GITHUB_NAMESPACE		?= seznam
+GITHUB_PROJECT			?= slo-exporter
+SLO_EXPORTER_VERSION	?= test
+OS				= linux
+ARCH			= amd64
+BINARY_PATH		= build/$(OS)-$(ARCH)/slo_exporter
+src_dir			:= $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 
+.PHONY: all
 all: lint build test-and-coverage
 
+.PHONY: build
 build:
-	GOOS=$(OS) CGO_ENABLED=0 go build -a -ldflags "-X 'main.buildVersion=${SLO_EXPORTER_VERSION}' -X 'main.buildRevision=${CI_COMMIT_SHA}' -X 'main.buildBranch=${CI_COMMIT_BRANCH}' -X 'main.buildTag=${CI_COMMIT_TAG}' -extldflags '-static'" -o slo_exporter $(src_dir)/cmd/slo_exporter.go
+	GOOS=$(OS) GOARCH=$(ARCH) CGO_ENABLED=0 go build -a -ldflags "-X 'main.buildVersion=${SLO_EXPORTER_VERSION}' -X 'main.buildRevision=${CIRCLE_SHA1}' -X 'main.buildBranch=${CIRCLE_BRANCH}' -X 'main.buildTag=${CIRCLE_TAG}' -extldflags '-static'" -o $(BINARY_PATH) $(src_dir)/cmd/slo_exporter.go
 
+.PHONY: docker-build
+docker-build:
+	docker build -t $(DOCKER_IMAGE_REPO):$(SLO_EXPORTER_VERSION) .
+	docker run --rm $(DOCKER_IMAGE_REPO):$(SLO_EXPORTER_VERSION) --help
+
+.PHONY: docker-push
+docker-push:
+	docker push $(DOCKER_IMAGE_REPO):$(SLO_EXPORTER_VERSION)
+
+.PHONY: github-release
```

Comment:
I don't think we need this in Makefile actually, IMHO it's OK to run the script directly in the CI job definition WDYT?

Reply:
Let's keep it as it is, at least for now.